lib/vscode/src/vs/server/node/channel.ts

```diff
@@ -866,6 +867,20 @@ export class TerminalProviderChannel implements IServerChannel<RemoteAgentConnec
 		throw new Error('not implemented');
 	}
 
+	private async listProcesses(): Promise<IProcessDetails[]> {
+		// TODO: args.isInitialization. Maybe this is to have slightly different
```

Comment:
I've seen names on todos occasionally but wasn't sure why. What's the benefit of adding a name? If it's to figure out who wrote the comment wouldn't that be better suited to a `git blame`?

Reply:
Then it assigns ownership to the todo (something I've practiced at previous places).

But if I go and do the next vscode update and it touches this file, won't `git blame` show me and not you?

Reply:
Yup if you modify that line and you only look at the last modified that would be the case. You'd want to go back to when it was added. There isn't much you get by adding information to comments that already exist in the commits or code IMO.

But even though you can find the original author, in that case I'd argue that if you move or edit a comment or the code around it then it might make sense to inherit ownership of the todo. Someone who last modified the code might be in a better position to complete the todo than the person who first added it (who might have done it years ago, or doesn't work here anymore, or is on another team, or might not recognize the code if it's been through a few iterations by other authors, etc).

To me adding the name also supports the idea that "someone else will do it" and then no one will do it. :stuck_out_tongue: For example, I don't have the bandwidth to look into any of these so it'd be unfortunate if someone reading these thought I was going to get around to it.

If you are definitely going to do it and don't want anyone else to take it then I think it makes sense to drop a name.

But if a task needs to be owned/completed by a specific individual I think creating and assigning a GitHub issue might make more sense.

Reply:
> If you are definitely going to do it and don't want anyone else to take it then I think it makes sense to drop a name.

Totally agree! I think this should be the rule of thumb then ^

> But if a task needs to be owned/completed by a specific individual I think creating and assigning a GitHub issue might make more sense.

Agreed. An issue will always be the most reliable way to ensure work gets done.